app/assets/javascripts/discourse/templates/preferences/account.hbs

```diff
@@ -80,32 +80,38 @@
 
     {{passwordProgress}}
   </div>
+</div>
 
+<div class="control-group pref-second-factor">
+  <label class="control-label">{{i18n 'user.second_factor.title'}}</label>
+  {{#unless model.second_factor_enabled}}
+    <label>
+        {{i18n 'user.second_factor.short_description'}}
+    </label>
+  {{/unless}}
   <div class="controls pref-second-factor">
-    {{#if model.second_factor_enabled}}
-      {{i18n 'user.second_factor.disable'}}
-    {{else}}
-      {{i18n 'user.second_factor.enable'}}
-    {{/if}}
-
     {{#if isCurrentUser}}
-      {{#link-to "preferences.second-factor" class="btn btn-default btn-small btn-icon pad-left no-text"}}
-        {{d-icon "pencil-alt"}}
-      {{/link-to}}
+      {{#if model.second_factor_enabled}}
+        {{#link-to "preferences.second-factor" class="btn btn-default"}}
+          {{d-icon "unlock"}} {{i18n 'user.second_factor.disable'}}
```

Comment:
If feel we should wrap i18n part in span here, I really don't like text floating around with no parent container.